package.json

```diff
@@ -1,33 +1,38 @@
 {
   "name": "@fusebit/monaco-jsonforms",
-  "version": "1.0.8",
+  "version": "1.0.9",
   "main": "dist/cjs/index.js",
   "module": "dist/esm/index.js",
   "devDependencies": {
+    "@material-ui/core": "^4.12.4",
```

Comment:
Make sure that this addition doesn't cause any breakages on our existing .html files - I _think_ it should be okay, but I barely understand what's going on in the best of times, so a manual validation would be wise.